tests/budget_accounting_test.py

```diff
@@ -95,13 +101,43 @@ class ComputeBudgetTestCase:
             mechanisms: []
 
         testcases = [
+            ComputeBudgetTestCase(
+                name="generic",
+                epsilon=0.22999925338484556,
+                delta=1e-5,
+                mechanisms=[
+                    ComputeBudgetMechanisms(
+                        1,
+                        6.41455078125,
+                        MechanismType.GENERIC,
+                        1,
+                        1,
+                        expected_mechanism_epsilon=0.2204717161227536,
+                        expected_mechanism_delta=9.585757904781109e-06)
+                ],
+                expected_pipeline_noise_std=6.41455078125),
+            ComputeBudgetTestCase(
+                name="generic_multiple",
+                epsilon=0.6599974547358093,
+                delta=1e-5,
+                mechanisms=[
+                    ComputeBudgetMechanisms(
+                        3,
```

Comment:
Nit: please add named arguments for ComputeBudgetMechanisms

Reply:
Fixed! 👍